web/elm/src/Build/Output/Output.elm

```diff
@@ -100,60 +100,37 @@ handleStepTreeMsg action model =
 
 planAndResourcesFetched :
     Concourse.BuildId
-    -> Result Http.Error ( Concourse.BuildPlan, Concourse.BuildResources )
+    -> ( Concourse.BuildPlan, Concourse.BuildResources )
```

Comment:
...Hrmmm. I get that the tuple is easier to pass from the caller, it just seems... odd.